### ISSUE TITLE
Your package isn't compatible with scikit-learn 1.0.1

### ISSUE THREAD

**Issue by shayandavoodii** [8080labs/ppscore#67]
Please make related changes in your package to be compatible with `scikit-learn 1.0.1`. I installed your package, pip uninstalled the sklearn, and installed `scikit-learn-0.24.2`!!! This is too old!

```python
>>>pip install -U ppscore
Collecting scikit-learn<1.0.0,>=0.20.2
  Downloading scikit_learn-0.24.2-cp39-cp39-win_amd64.whl (6.9 MB)
     |████████████████████████████████| 6.9 MB 939 kB/s
```

**Comment by marcessz** [comment]
I am having the same problem :((((

ContextualVersionConflict: (scikit-learn 1.1.2, Requirement.parse('scikit-learn<1.0.0,>=0.20.2'), {'ppscore'})

**Comment by 8080labs** [comment]
This is fixed with ppscore 1.3.0
